doc/base/classes.xml

```diff
@@ -2967,20 +2969,28 @@
 	</methods>
 	<members>
 		<member name="animation" type="String" setter="set_animation" getter="get_animation" brief="">
+			The current animation's name.
 		</member>
 		<member name="centered" type="bool" setter="set_centered" getter="is_centered" brief="">
+			If true, offset is relative to the sprite's center. Top-left corner otherwise.
 		</member>
 		<member name="flip_h" type="bool" setter="set_flip_h" getter="is_flipped_h" brief="">
+			If true, horizontally flips the sprite.
 		</member>
 		<member name="flip_v" type="bool" setter="set_flip_v" getter="is_flipped_v" brief="">
+			If true, vertically flips the sprite.
 		</member>
 		<member name="frame" type="int" setter="set_frame" getter="get_frame" brief="">
+			The current frame number.
 		</member>
 		<member name="frames" type="SpriteFrames" setter="set_sprite_frames" getter="get_sprite_frames" brief="">
+			How many frames the AnimatedSprite has.
```

Comment:
This variable's type is [SpriteFrames] not int.  Its value is the [SpriteFrames] Resource.